src/components/modals/PaperWallet/PaperWallet.vue

```diff
@@ -84,11 +84,11 @@ export default class PaperWallet extends Vue {
             let wallet: MnemonicWallet = this.$store.state.activeWallet
             if (!wallet) return '-'
 
-            let key = wallet.externalHelper.getKeyForIndex(0)
+            let key = wallet.getCurrentAddressPlatform()
```

Comment:
We used the same thing in walletTitle function, what does getCurrentAddressPlatform() return ??